matrix/common/etl/dcp_zarr_store.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Class for working with expression matrices serialized in the DCP."""
+
+import glob
+import json
+import os
+from collections import MutableMapping
```

Comment:
I get a `DeprecationWarning` on this in 3.7, saying it should be `from collections.abc ...`

Reply:
Good catch